bench-vortex/src/bin/clickbench.rs

```diff
@@ -208,7 +208,14 @@ fn main() -> anyhow::Result<()> {
         .targets
         .iter()
         .any(|t| t.engine() == Engine::DuckDB)
-        .then(|| ddb::build_and_get_executable_path(&args.duckdb_path, false));
+        .then(|| {
+            let path = ddb::get_executable_path(&args.duckdb_path);
+            // If the path is to vortex duckdb, try to rebuild
```

Comment:
I think this comment is a little misleading. It's more like if the path is not set at all.